internal/manifest/referencevalues.go

```diff
@@ -25,6 +31,13 @@ type ReferenceValues struct {
 	BareMetalTDX *BareMetalTDXReferenceValues `json:"bareMetalTDX,omitempty"`
 }
 
+// PlatformRuntimeMapping is a 1:1 mapping of platform names to their runtime component hashes.
+type PlatformRuntimeMapping struct {
+	AKS          string `json:"aks,omitempty"`
+	BareMetalTDX string `json:"bareMetalTDX,omitempty"`
+	BareMetalSNP string `json:"bareMetalSNP,omitempty"`
```

Comment:
I don't like that this is silently introducing sort of a "PlatformGroup" mechanism without a clean notation.